package/src/components/QuestionPromptText.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+import { Typography } from '@mui/material';
+
+interface QuestionPromptTextProps {
+    prompt: string;
+    required: boolean;
+}
+
+const QuestionPromptText: FC<QuestionPromptTextProps> = ({ prompt, required }) => (
+    <Typography variant="body1" gutterBottom>
+        {prompt} {required && <span style={{ color: 'red' }}>*</span>}
+    </Typography>
+);
+
+export default QuestionPromptText;
```

Comment:
I really like the idea of separating this logic. Can we move this into a different folder? The components folder is meant for components in the toolbox to be added to forms, not necessarily sections of logic within each field

A new folder may need to be created but I really like this idea

Reply:
it looks like all the components that are in the toolbox are part of the `elements` folder and I think I created this new folder called `components`, but it can be confusing so I renamed the folder which we can keep all our components not used in the toolbox to `helperComponents`